### ISSUE TITLE
Add libhtp to vcpkg

### ISSUE THREAD

**Issue by M010** [OISF/libhtp#406]
It would be a great idea to add libhtp to vcpkg library

**Comment by catenacyber** [comment]
Why so ? Could you elaborate ?
For Suricata's usage, the plan is to switch to a rust version cf https://github.com/cccs-rtmorti/libhtp-rs

**Comment by M010** [comment]
@catenacyber Yeah, I can! Suricata is not the only project that uses libhtp. And for a lot of people, vcpkg and conan are becoming the standard for development, like cargo in Rust. So, after Suricata switches to the Rust version, what is your vision about this C++ library?

**Comment by catenacyber** [comment]
> Suricata is not the only project that uses libhtp.

We do not have much vision about other projects. Would you know one ?

> what is your vision about this C++ library?

I think it will end up archived as read-only after some years...

**Comment by victorjulien** [comment]
Closing in preparation of archiving the repo, see #467.
